pkgs/development/python-modules/mwtypes/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, nose
+, jsonable
+}:
+
+buildPythonPackage rec {
+  pname = "mwtypes";
+  version = "0.3.2";
+
+  src = fetchPypi {
```

Comment:
why not `fetchFromGitHub`?

Reply:
Because unfortunately there are no releases/tags on the repo.